subsuper-proposal.tex

```diff
@@ -163,7 +163,7 @@ \section{Required Subscript/Superscript Characters}
 
 \subsection{Relationship to existing sub/superscript letters}
 
-A subset of Latin and Greek letters already have sub/superscript variants in Unicode.  For example, `$^\mathrm{a}$' is U+1D43 and `$_\mathrm{a}$' is U+2090, while `$^\mathrm{A}$' is U+1D2C but there is no subscript `A' character.   However, these codepoints seem to have been added piecemeal into the Unicode standard for a variety of semantic meanings unrelated to their usage in mathematical notation (e.g. `$^\mathrm{a}$' is in the Latin-1 Supplement block, whereas `$^\mathrm{k}$' is in the Phonetic Extensions block).   (Accordingly, they are rendered with inconsistent baselines and kerning in many fonts.) Therefore we recommend that these existing characters \textbf{should not be canonically equivalent} to the proposed new mathematical sub/superscript characters (whether implemented by new codepoints or by combining characters.
+A subset of Latin and Greek letters already have sub/superscript variants in Unicode.  For example, `$^\mathrm{a}$' is U+1D43 and `$_\mathrm{a}$' is U+2090, while `$^\mathrm{A}$' is U+1D2C but there is no subscript `A' character.   However, these codepoints seem to have been added piecemeal into the Unicode standard for a variety of semantic meanings unrelated to their usage in mathematical notation (e.g. `$^\mathrm{a}$' is in the Latin-1 Supplement block, whereas `$^\mathrm{k}$' is in the Phonetic Extensions block). Accordingly, they are rendered with inconsistent baselines and kerning in many fonts. Therefore we recommend that these existing characters \textbf{should not be canonically equivalent} to the proposed new mathematical sub/superscript characters (whether implemented by new codepoints or by combining characters).
 
```

Comment:
There is nothing wrong with putting a sentence in parentheses, so I'm not sure why this is a "fix".

The comment about fonts really is an aside, because font rendering is not under the control of the Unicode standard.